rm_base:
  bus:
    - can0
  loop_frequency: 1000
  cycle_time_error_threshold: 0.001
  # Configurations of the actuators
  actuators:
    joint1_motor:
      bus: can0
      id: 0x201
      type: rm_3508
    joint2_motor:
      bus: can0
      id: 0x002
      type: cheetah
    joint3_motor:
      bus: can0
      id: 0x001
      type: cheetah
    left_wrist_motor:
      bus: can0
      id: 0x203
      type: rm_2006
    right_wrist_motor:
      bus: can0
      id: 0x202
      type: rm_2006